packages/components/src/button/index.jsx

```diff
@@ -12,7 +12,7 @@ import './style.scss';
 
 export default class Button extends PureComponent {
 	static propTypes = {
-		bare: PropTypes.bool,
+		plain: PropTypes.bool,
```

Comment:
Should we introduce legacy support for `bare` for a couple of versions? Just so we'd not break any external consumers of this package that use the old prop. 

If you're in favor, I'd envision for this legacy support:
* If `bare` is provided, but `plain` is not provided, `plain` uses the `bare value`.
* If `bare` is used, trigger a warning.
* Add a comment to remove the `bare` support after several versions.

In any case, we should also document this in the CHANGELOG of the package.

What do you think?

Reply:
`bare` was added just a few days ago in #45386. It didn't exist before then, and the components package wasn't published since then. We can still rename it to `plain` without any special concerns.

Reply:
Ah, TIL. Let's add a CHANGELOG entry then and we should be good 👍